src/client/src/orangehrmRecruitmentPlugin/pages/SaveCandidate.vue

```diff
@@ -0,0 +1,198 @@
+                :placeholder="$t('general.type_here')"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                v-model="candidate.application"
+                :label="$t('recruitment.date_of_application')"
+                type="date"
+                :placeholder="$t('general.type_here')"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item class="orangehrm-save-candidate-page-full-width">
+              <oxd-input-field
+                v-model="candidate.notes"
+                :label="$t('general.notes')"
+                type="textarea"
+                :placeholder="$t('general.type_here')"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item
+              class="orangehrm-save-candidate-page-full-width orangehrm-save-candidate-page-grid-checkbox"
+            >
+              <oxd-input-field
+                v-model="candidate.keep"
+                type="checkbox"
+                :label="$t('recruitment.content_to_keep_data')"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button display-type="ghost" :label="$t('general.cancel')" />
+          <submit-button />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import FullNameInput from '@/orangehrmPimPlugin/components/FullNameInput';
+import {shouldNotExceedCharLength} from '@/core/util/validation/rules';
+import VacancyDropdown from '@/orangehrmRecruitmentPlugin/components/VacancyDropdown';
+import SubmitButton from '@/core/components/buttons/SubmitButton';
+import {required, validEmailFormat} from '@/core/util/validation/rules';
+
+import {APIService} from '@ohrm/core/util/services/api.service';
+export default {
+  name: 'SaveCandidate',
+  components: {SubmitButton, VacancyDropdown, FullNameInput},
+  setup() {
+    const http = new APIService(
+      'https://884b404a-f4d0-4908-9eb5-ef0c8afec15c.mock.pstmn.io',
+      'recruitment/api/candidate',
+    );
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      candidate: {
+        firstName: null,
+        middleName: '',
+        lastName: '',
+        email: '',
+        contactNumber: '',
+        resume: '',
+        vacancy: null,
+        keywords: '',
+        application: '',
+        notes: '',
+        keep: '',
+      },
+      rules: {
+        firstName: [required, shouldNotExceedCharLength(30)],
+        lastName: [required, shouldNotExceedCharLength(30)],
```

Comment:
middle name length validation
Contact Number validation
resume validation
keywords validation
notes validation
date of application validation